tests/lightgbm/test_lightgbm_autolog.py

```diff
@@ -423,3 +429,42 @@ def test_lgb_autolog_continues_logging_even_if_signature_inference_fails(tmpdir)
     assert data is not None
     assert "run_id" in data
     assert "signature" not in data
+
+
+@pytest.mark.large
+def test_lgb_autolog_configuration_options(bst_params):
+    iris = datasets.load_iris()
+    X = pd.DataFrame(iris.data[:, :2], columns=iris.feature_names[:2])
+    y = iris.target
+
+    with mlflow.start_run() as run:
+        mlflow.lightgbm.autolog(log_input_example=False, log_model_signature=False)
+        dataset = lgb.Dataset(X, y)
+        lgb.train(bst_params, dataset)
+    model_conf = get_model_conf(run.info.artifact_uri)
+    assert "saved_input_example_info" not in model_conf.to_dict()
```

Comment:
Nit: I think this can be simplified as `model_conf.saved_input_example_info is None`. This is the approach we take in other test suites.

Reply:
Personally I think this way is slightly cleaner, because it makes it clear that you don't expect that field to be there at all. Versus the one above either `saved_input_example_info` could not exist, or it could have the value actually be none.

What do you think?